repository/user.repository.js

```diff
@@ -0,0 +1,14 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+
+async function getUser(username, password)
```

Comment:
```suggestion
const getUser = async (username, password) => 
```
Cela ne change rien fonctionnellement mais si on souhaite être iso avec le reste du code, mieux vaut éviter la syntaxe `function`